frontend/apps/reader/modules/readersearch.lua

```diff
@@ -21,12 +37,26 @@ function ReaderSearch:init()
     self.ui.menu:registerToMainMenu(self)
 end
 
+local help_text = [[Here you can search text or a regular expression (ECMAScript syntax).
```

Comment:
I'm not sure how useful mentioning ES syntax is. I suppose it defines a few unique extensions?

http://www.akenotsuki.com/misc/srell/en/#syntax

Reply:
I think we should give some hint here, which syntax can be used as there are quite some different implementations: ECMAScript, awk, grep, egrep, basic, extended srell ...
Best would be a the link http://www.akenotsuki.com/misc/srell/en/#syntax (at the bottom of the info) 
....
...
`The ECMA like syntax can be found on http://www.akenotsuki.com/misc/srell/en/#syntax.`

Reply:
My point is primarily, imagine you're a regular user who just stumbled on this mysterious "regular expression" thing.

The explanatory text starts off with "here's a paraphrase of what you already knew, namely that it's a search dialog for 'regular expressions' followed by some obfuscating gibberish." ;-)

This is not a good opener.

Reply:
Maybe something like:
```Regular expressions allow you to search for a matching pattern in a text. The simplest pattern is a simple sequence of characters, such as `James Bond`. There are many different varieties of regular expressions, but we support the ECMAScript syntax. The basics will be explained below.```